docs/documents/powerbi.md

```diff
@@ -0,0 +1,35 @@
+PowerBI dashboard data
+
+The powerBI db backup pipeline was migrated from the old paas environment to the new aks environment
+
+the old pipeline was implemented using azure devops classic release pipeline.
+The new pipeline is also implemented using the classic azure devops release pipeline and it is located here
+https://dev.azure.com/dfe-ssp/S105-School-Experience/_release?_a=releases&view=mine&definitionId=66
+
+the pipeline works by doing a backup of the kubernetes application database in aks. The database is accessed byport forwarding using Konduit.
+
+There are two set of credentials used in the pipeline namely admin and perfappuser
+the admin credentials is used in deleting and recreating the perf analysis database
```

Comment:
The password is set on the Azure databse server and copied to Azure devops pipeline variables